root/release/edit/information.tt

```diff
@@ -249,7 +249,7 @@
       </tr>
 
       <tr>
-        <td><label for="comment">[% add_colon(l('Disambiguation')) %]</label></td>
+        <td><label for="comment">[% l('Disambiguation:') %]</label></td>
```

Comment:
Oops, why do you think it is pointless?
The purpose of `add_colon` (name can be changed if needed) is twofold:
* to prevent duplicates between labels depending on presentation (e.g. `Name` and `Name:`)
* to uniformize localization of labels ending with a colon `Foo:`

Reply:
I have two reasons for this, as well :)

1) Colon is not a universal symbol in this function (nor formatted universally alike), and it would be better if translators could replace it, if needed.
See these two links for examples:
https://en.wikipedia.org/wiki/Punctuation#Other_languages
https://en.wikipedia.org/wiki/Colon_(punctuation)#End_of_sentence_or_verse

2) In my language, Estonian (and surely some other languages, too), it is sometimes needed to translate a word slightly differently (e.g. using a different case ending) depending on whether it's used as a label for a text field or as title of something.

Reply:
>     1. Colon is not a universal symbol in this function (nor formatted universally alike), and it would be better if translators could replace it

Totally agree, and translators can fortunately replace it, see https://github.com/metabrainz/musicbrainz-server/blob/v-2020-03-31/po/mb_server.pot#L6994-L6998

>     2. In my language, Estonian (and surely some other languages, too), it is sometimes needed to translate a word slightly differently (e.g. using a different case ending) depending on whether it's used as a label for a text field or as title of something.

Can you please provide examples? I’m inclined to believe you, that is not the issue. Because we intentionally replaced colon strings with `addColon` calls until now, we need to convincingly document the reasoning and create guidelines for future coders to add properly localizable strings rather than to do the same mistake again.

Reply:
We should probably still replace them, in any case - it's just we might need to add context in some cases.

Reply:
> Totally agree, and translators can fortunately replace it, see https://github.com/metabrainz/musicbrainz-server/blob/v-2020-03-31/po/mb_server.pot#L6994-L6998

Okay, that makes it somewhat better at least.

> > ```
> > 2. In my language, Estonian (and surely some other languages, too), it is sometimes needed to translate a word slightly differently (e.g. using a different case ending) depending on whether it's used as a label for a text field or as title of something.
> > ```
> 
> Can you please provide examples?

Ended: → Lõpp: ("the end" - not ideal, but had to find something that would work for both artist, label and area entities)
Ended → Pole enam aktiivselt kasutusel ("not in active use anymore", admittedly a rather loose translation, but again, that's what worked for all the various contexts of this string)

Note: → Pane tähele: (verb)
Note → Märkus (noun)

Votes for: → Poolthääli: ("[number of] favourable votes")
Votes for → Poolt ("in favour")

Votes against: → Vastuhääli:
Votes against → Vastu

These are just from a quick look, and only from server.po. Sure, there were also many otherwise identical strings where the colon really was the only difference, but I still stand by the idea that it's easier for translators if we can quickly see the _complete_ string, including any trailing colons, in order to properly localize it. Especially with short phrases (1-3 words), my experience is that it's easier to translate 3 separate versions of a string (presuming one can understand their context) than to try and find a translation that _kind of works_ for all the various contexts lumped into 1 source string :)

Reply:
> We should probably still replace them, in any case - it's just we might need to add context in some cases.

It's hard to know beforehand what needs context - and even what kind of context makes sense to different persons, as is apparent in the comments here.